annotationProcessor/src/main/java/org/corfudb/annotations/ObjectAnnotationProcessor.java

```diff
@@ -0,0 +1,642 @@
+package org.corfudb.annotations;
+
+import com.google.common.collect.ImmutableMap;
+import com.squareup.javapoet.*;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import org.corfudb.runtime.object.*;
+
+import javax.annotation.processing.*;
+import javax.lang.model.SourceVersion;
+import javax.lang.model.element.*;
+import javax.lang.model.type.DeclaredType;
+import javax.lang.model.type.TypeKind;
+import javax.lang.model.type.TypeMirror;
+import javax.lang.model.util.Elements;
+import javax.lang.model.util.Types;
+import javax.tools.Diagnostic;
+import java.io.IOException;
+import java.lang.reflect.Field;
+import java.util.*;
```

Comment:
This is what intelliJ defaults to when about 5 or more imports come from a particular package. Is it really necessary to do this? Who reads the import section anyway?